### PR TITLE
Run scripts for voiced dialogue

### PR DESCRIPTION
**Pull request by Allofich**
For https://bugs.openmw.org/issues/3787.

### CODE REVIEW

File: apps/openmw/mwdialogue/dialoguemanagerimp.cpp [hunk 1]
```diff
@@ -647,6 +647,9 @@ namespace MWDialogue
             if (!info->mSound.empty())
                 sndMgr->say(actor, info->mSound);
         }
+
+        mActor = actor;
```
Comment (scrawl) [inline apps/openmw/mwdialogue/dialoguemanagerimp.cpp]: It's incorrect to overwrite mActor. That is the actor that the dialogue window is opened for, but ```say``` is completely independent of that window. I suggest to add a new parameter (actor) to executeScript.

Comment (Allofich) [inline apps/openmw/mwdialogue/dialoguemanagerimp.cpp]: Fixed.

File: apps/openmw/mwdialogue/dialoguemanagerimp.cpp [hunk 2]
```diff
@@ -647,6 +647,9 @@ namespace MWDialogue
             if (!info->mSound.empty())
                 sndMgr->say(actor, info->mSound);
         }
+
+        if (!info->mResultScript.empty())
```
Comment (MiroslavR) [inline apps/openmw/mwdialogue/dialoguemanagerimp.cpp]: Info is not checked for NULL, which is causing crashes on my end. Please fix it.

Comment (Allofich) [inline apps/openmw/mwdialogue/dialoguemanagerimp.cpp]: Sorry, new PR up.

**Pull request merged**